src/Pool.sol

```diff
@@ -322,6 +351,39 @@ abstract contract Pool {
         }
     }
 
+    function _collectCycles(address receiverAddr, uint64 count)
```

Comment:
I think this is already outdated and we removed the `lastFundsPerCycle`

Reply:
Outdated